app/Http/Controllers/EpisodeManager.php

```diff
@@ -23,9 +25,39 @@ class EpisodeManager extends Controller
     */
     public function index()
     {
-        $episodes = Episode::all();
 
-        return view('dashboard/pages/view_episodes', compact('episodes'));
+        $user                           = User::all();
+        $online_user                    = $user->where('active', 1)->count();
+        $offline_user                   = $user->where('active', 0)->count();
+        $numbers_of_users_on_suyabay    = $user->count();
+
+        $episodes                       = Episode::get();
+        $active_episode                 = $episodes->where('status', 1);
+        $pendding_episode               = $episodes->where('status', 0);
+
+        $channels                       = Channel::get();
+
+        $data = 
```

Comment:
The data returned by the index method are collection of information needed to populate the dashboard area